Soujanya2004/wanderlust-2024
# added customized pages to the footer quick links

Fixes #98

Adds customized pages to the footer  "Quick Links".

### Changes
Updated the footer quick links:
Privacy Policy
Terms & Conditions
Ensured that each link directs users to the correct page for seamless navigation.

### Changes
List the detailed changes made in this PR.
- Added a new feature to ...
- Refactored the ...
- Fixed a bug in ...

### Testing Instructions
* Testing Instructions
* Pull this branch.
* Run npm install (if necessary).
* Navigate to the website's footer section.
* Verify that the "Quick Links" section contains the new links.
* Click each link to ensure it directs to the correct page.

### Screenshots (if applicable)
![localhost_8000_privacy](https://github.com/user-attachments/assets/9b3c1151-2e80-4d8f-a44a-d5603fd19c5a)
![localhost_8000_terms](https://github.com/user-attachments/assets/0ee05cb1-9925-4082-a7d9-13d3f75ba672)


### Checklist
 I have tested the new footer links for correct functionality.
 I have performed a self-review of my code.
 I have commented the code where necessary.
 No new warnings were introduced.
